modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -97,14 +97,18 @@ image::manage-settings/node-availability.png["The Node Availability panel",548,a
 
 The following checkboxes are provided:
 
-* *Auto-failover after _x_ seconds for up to _y_ node*: After the timeout period set here as _x_ seconds has elapsed, an unresponsive or malfunctioning node is failed over, provided that the limit on actionable events set here as _y_ has not yet been reached.
+* *Auto-failover after _x_ seconds for up to _y_ node*: After the timeout period set here as _x_ seconds has elapsed, an unresponsive or malfunctioning node is failed over, provided that the limit on actionable events set here as _y_ (with the default value of 1) has not yet been reached.
 Data replicas are promoted to active on other nodes, as appropriate.
 This feature can only be used when three or more nodes are present in the cluster.
 The number of seconds to elapse is configurable: the default is 120; the minimum permitted is 5; the maximum 3600.
 This option is selected by default.
+
 * *Auto-failover for sustained data disk read/write failures after _z_ seconds*: After the timeout period set here as _z_ seconds has elapsed, a node is failed over if it has experienced sustained data disk read/write failures.
 The timeout period is configurable: the default length is 120 seconds; the minimum permitted is 5; the maximum 3600.
 This checkbox can only be checked if *Auto-failover after _x_ seconds for up to _y_ node* has also been checked.
+This option is selected by default.
+
+* *Preserve durable writes*: If this checkbox is checked (which is the default), and the node to be automatically failed over is a Data Service node that hosts data for a bucket whose Minimum Durability Level is greater than *none*; auto-failover occurs only if xref:learn:data/durability.adoc#majority[Majority] can be maintained across the nodes on which the bucket's data resides.
```

Comment:
> auto-failover occurs only if xref:learn:data/durability.adoc#majority[Majority] can be maintained across the nodes on which the bucket's data resides.
 
This is incorrect. A majority is the required number of nodes to accept and process a durable write. This check will still allow the failover of nodes such that a majority may not be possible post-failover.  The intent is, quite simply, to avoid failing over a node if it could result in the loss of an acknowledged and committed durable write. This comment explains the mechanics of the change, which hopefully helps to explain the intent too - https://github.com/couchbase/ns_server/blob/cd0d04312f5f2dec6f411c7f5002b16f21f09bb9/src/failover.erl#L547-L599. Please let me know if that's not clear.